app/javascript/packages/rails-i18n-webpack-plugin/extract-keys-webpack-plugin.js

```diff
@@ -0,0 +1,108 @@
+const sources = require('webpack-sources');
+
+/**
+ * Webpack plugin name.
+ *
+ * @type {string}
+ */
+const PLUGIN = 'ExtractKeysWebpackPlugin';
+
+/**
+ * Regular expression matching translation calls.
+ *
+ * @see https://github.com/glebm/i18n-tasks/blob/v0.9.34/lib/i18n/tasks/scanners/pattern_scanner.rb#L15
+ *
+ * @type {RegExp}
+ */
+const TRANSLATE_CALL = /(?:^|[^\w'-])(?:I18n\.)?t\(\s*['"](.+?)['"]/g;
+
+/**
+ * Given an original file name and suffix, returns a modified file name with the suffix injected
+ * prior to the original file extension.
+ *
+ * @param {string} filename
+ * @param {string} suffix
+ *
+ * @return {string}
+ */
+function getAdditionalAssetFilename(filename, suffix) {
```

Comment:
what if we called this `locale` instead of suffix? It helped the specs make more sense to me

Reply:
> what if we called this `locale` instead of suffix? It helped the specs make more sense to me

Yeah, that might make more sense.

For context, this is likely an artifact of the way the implementation evolved, where `ExtractKeysWebpackPlugin` was somewhat generic about what it expected subclasses to produce as additional assets for the keys it discovered. But I think it's moved in enough a direction where it's pretty unlikely this suffixing behavior would be used for anything other than adding the locale 🤷

Reply:
> what if we called this `locale` instead of suffix? It helped the specs make more sense to me

Renamed in a8f36af.